tutorial/10_key_features/005_visualization.py

```diff
@@ -17,6 +20,9 @@
 from sklearn.model_selection import train_test_split
 
 import optuna
+
+# You can use Matplotlib instead of Plotly for visualization by simply replacing `optuna.visualization` to
```

Comment:
Sorry, one more thing. According to Grammarly,
```suggestion
# You can use Matplotlib instead of Plotly for visualization by simply replacing `optuna.visualization` with
```